src/cmd/statsrelay.rs

```diff
@@ -33,6 +34,86 @@ struct Options {
     pub threaded: bool,
 }
 
+/// The main server invocation, for a given configuration, options and stats
+/// scope. The server will spawn any listeners, initialize a backend
+/// configuration update loop, as well as register signal handlers.
+async fn server(scope: stats::Scope, config: Config, opts: Options) {
+    let backends = backends::Backends::new(scope.scope("backends"));
+
+    let backend_reloads = scope.counter("backend_reloads").unwrap();
+
+    let (sender, tripwire) = Tripwire::new();
+    let mut run: FuturesUnordered<_> = config
+        .statsd
+        .servers
+        .iter()
+        .map({
+            |(server_name, server_config)| {
+                let name = server_name.clone();
+                statsd_server::run(
+                    scope.scope("statsd_server").scope(server_name),
+                    tripwire.clone(),
```

Comment:
I think this clone's superfluous, since the tripwire future's only ever used here.

Reply:
The stream_cancel tripwire has to move, and we can have > 1 server, so this clone is needed